common/eqemu_logsys_log_aliases.h

```diff
@@ -1028,6 +1038,12 @@
 #define LogDynamicZonesDetail(message, ...) do {\
 } while (0)
 
+#define LogClientList(message, ...) do {\
```

Comment:
`LogCheat` stuff missing here (not a bug from this branch, just noticed it missing if BUILD_LOGGING is disabled)

Reply:
Yeah I'll add them to compensate for what was missing from the cheat PR